Dangerfile

```diff
@@ -9,5 +9,6 @@ swiftlint.lint_files inline_mode: true
 
 # SwiftFormat
 swiftformat.binary_path = "bin/swiftformat"
-swiftformat.additional_args = "--config .swiftformat --swiftversion 4.2"
+swiftformat.additional_args = "--config .swiftformat --swiftversion 5"
```

Comment:
Not sure why this was still on `4.2`.

Also added our two generated files to be excluded by Danger here to avoid SwiftFormat failing saying that it has nothing to format.